wagtailio/urls.py

```diff
@@ -58,6 +58,7 @@
 ):
     urlpatterns += [
         path("pattern-library/", include("pattern_library.urls")),
+        path("test500", test_500_error),
```

Comment:
Hey @zerolab, 

how do you feel about a testing view for the 500 error page?

Reply:
👍 for the approach